### ISSUE TITLE
10.0.19041.1320 not supported Win10 X64

### ISSUE THREAD

**Issue by zekecarter** [stascorp/rdpwrap#1574]
Windows 10 Home
Version: 20H2
OS Build: 19042.1320
[termsrv.zip](https://github.com/stascorp/rdpwrap/files/7422456/termsrv.zip)

![img](https://user-images.githubusercontent.com/83554130/138984952-6ddf827d-daa8-46ed-a231-7852b574ec81.png)

**Comment by mchotdograp** [comment]
same here

**Comment by docikstorm** [comment]
Me too on ver 10.0.19041.1320 but on Win 10 Pro x64.
Not work.
 :(

**Comment by docikstorm** [comment]
Thanks already working.

**Comment by mchotdograp** [comment]
> Thanks already working.

What? Please share

**Comment by sebaxakerhtc** [comment]
> 
> 
> > Thanks already working.
> 
> What? Please share

https://github.com/sebaxakerhtc/rdpwrap.ini/commit/be598440ddc0a264e1caf435b614f46f806f924a

**Comment by pypk1** [comment]
I just ran autoupdate.bat and it is still not working.

**Comment by docikstorm** [comment]
> I just ran autoupdate.bat and it is still not working.

Try again but run as administrator.

I too use autoupdate.bat, and it work.

![image](https://user-images.githubusercontent.com/43828429/139023753-61ab3f0e-f424-45c3-8bd4-c009a85516b8.png)

**Comment by sebaxakerhtc** [comment]
> I just ran autoupdate.bat and it is still not working.

Do you have my source in autoapdate.bat?

**Comment by greyfoxID** [comment]
Still not working in W10 x64, eventhough I've use autoupdate.bat. And I use administrator too.

I'm still wondering why, any idea?

**Comment by sebaxakerhtc** [comment]
> Still not working in W10 x64, eventhough I've use autoupdate.bat. And I use administrator too.
> 
> I'm still wondering why, any idea?

All works fine.
Download latest autoupdate.bat  with my source or add it manually
https://raw.githubusercontent.com/sebaxakerhtc/rdpwrap.ini/master/rdpwrap.ini

**Comment by greyfoxID** [comment]
> 
> 
> > Still not working in W10 x64, eventhough I've use autoupdate.bat. And I use administrator too.
> > I'm still wondering why, any idea?
> 
> All works fine. Download latest autoupdate.bat with my source or add it manually https://raw.githubusercontent.com/sebaxakerhtc/rdpwrap.ini/master/rdpwrap.ini

Working! Thank you so much!

**Comment by sebaxakerhtc** [comment]
> Working! Thank you so much!

You're welcome

**Comment by boltorg** [comment]
![10 0 19041 1320](https://user-images.githubusercontent.com/93392843/139456918-02d75981-01b7-4f4b-8b0d-1f60e0a973c8.jpg)

Made sure that you were a source before running autoupdate (your source was included). This is what I'm getting. If I manually update the entries to include 10.0.19041.1320 I receive the following:

![rdpwrapconf](https://user-images.githubusercontent.com/93392843/139457496-0edd3c3b-ee8c-4d40-8468-7c8120d42073.jpg)


Running Windows 10 Home Version 21H1 (19043.1320)

**Comment by everdominguez** [comment]
> > Still not working in W10 x64, eventhough I've use autoupdate.bat. And I use administrator too.
> > I'm still wondering why, any idea?
> 
> All works fine. Download latest autoupdate.bat with my source or add it manually https://raw.githubusercontent.com/sebaxakerhtc/rdpwrap.ini/master/rdpwrap.ini

Thanks, it works fine.

**Comment by affinityv** [comment]
> ![10 0 19041 1320](https://user-images.githubusercontent.com/93392843/139456918-02d75981-01b7-4f4b-8b0d-1f60e0a973c8.jpg)
> 
> Made sure that you were a source before running autoupdate (your source was included). This is what I'm getting. If I manually update the entries to include 10.0.19041.1320 I receive the following:
> 
> ![rdpwrapconf](https://user-images.githubusercontent.com/93392843/139457496-0edd3c3b-ee8c-4d40-8468-7c8120d42073.jpg)
> 
> Running Windows 10 Home Version 21H1 (19043.1320)

Your antivirus / antimalware might be interfering, you need to make exceptions to stop them killing rdp.  You may also have to import a good Windows [registry](https://raw.githubusercontent.com/affinityv/INI-RDPWRAP/master/Windows_10_Pro_64bit_RDS.reg) file if it's been clobbered by said interferers!

**Comment by LookingforRDPhelp** [comment]
> > I just ran autoupdate.bat and it is still not working.
> 
> Try again but run as administrator.
> 
> I too use autoupdate.bat, and it work.
> 
> ![image](https://user-images.githubusercontent.com/43828429/139023753-61ab3f0e-f424-45c3-8bd4-c009a85516b8.png)

everytime i try to use auto update bat it always just instantly closes it out do you know how to fix this?

**Comment by TheArgentinian** [comment]
please, explain step by step how to add a source

**Comment by zekecarter** [comment]
> please, explain step by step how to add a source

https://github.com/stascorp/rdpwrap/issues/1588#issuecomment-962456582

**Comment by boltorg** [comment]
Still no dice on my end... I've disabled my AV, uninstalled, reinstalled, made sure service was stopped before updating ini manually. Listener state is hard stuck on "Not Listening"

**Comment by TheArgentinian** [comment]
> 
> 
> Still no dice on my end... I've disabled my AV, uninstalled, reinstalled, made sure service was stopped before updating ini manually. Listener state is hard stuck on "Not Listening"

Did you reboot the computer? because that's what changed the listening status for me

**Comment by boltorg** [comment]
> > Still no dice on my end... I've disabled my AV, uninstalled, reinstalled, made sure service was stopped before updating ini manually. Listener state is hard stuck on "Not Listening"
> 
> Did you reboot the computer? because that's what changed the listening status for me

Yes indeed. I've restarted after making the regedit change, uninstalling, reinstalling, etc. AV is off for all of this process as well.

**Comment by greyfoxID** [comment]
> 
> 
> > > Still no dice on my end... I've disabled my AV, uninstalled, reinstalled, made sure service was stopped before updating ini manually. Listener state is hard stuck on "Not Listening"
> > 
> > 
> > Did you reboot the computer? because that's what changed the listening status for me
> 
> Yes indeed. I've restarted after making the regedit change, uninstalling, reinstalling, etc. AV is off for all of this process as well.

Did you "run as administrator"? Because sometimes it makes difference with or without it in my case.

**Comment by boltorg** [comment]
> > > > Still no dice on my end... I've disabled my AV, uninstalled, reinstalled, made sure service was stopped before updating ini manually. Listener state is hard stuck on "Not Listening"
> > > 
> > > 
> > > Did you reboot the computer? because that's what changed the listening status for me
> > 
> > 
> > Yes indeed. I've restarted after making the regedit change, uninstalling, reinstalling, etc. AV is off for all of this process as well.
> 
> Did you "run as administrator"? Because sometimes it makes difference with or without it in my case.


Yes, all operations ran as admin. This is what I've tried. Ran uninstall.bat in C:\Program Files\RDP Wrapper as admin and rebooted. Disabled Remote Desktop Services and ran merged registry file, rebooted. Disabled Remote Desktop Services and copied the following rdpwrap.ini. 

[rdpwrap.txt](https://github.com/stascorp/rdpwrap/files/7514267/rdpwrap.txt)

Restarted Remote Desktop Services and it will not listen. 

![image](https://user-images.githubusercontent.com/93392843/141151566-e9d3246c-74ff-4881-ac93-bbc077d7d2dd.png)

**Comment by zekecarter** [comment]
your rdpwrap.ini does not have the update for version 10.0.19041.1348, you should look for the update in these comments https://github.com/stascorp/rdpwrap/issues/1595#issuecomment-964643800

**Comment by boltorg** [comment]
> your rdpwrap.ini does not have the update for version 10.0.19041.1348, you should look for the update in these comments [#1595 (comment)](https://github.com/stascorp/rdpwrap/issues/1595#issuecomment-964643800)

Yup that solved it, thanks for the help in the thread.

**Comment by MenghsuanLiu** [comment]
Dear Expert
How can I do without Internet?

**Comment by amshira** [comment]
> > Still not working in W10 x64, eventhough I've use autoupdate.bat. And I use administrator too.
> > I'm still wondering why, any idea?
> 
> All works fine. Download latest autoupdate.bat with my source or add it manually https://raw.githubusercontent.com/sebaxakerhtc/rdpwrap.ini/master/rdpwrap.ini

that works for me 
thanks a lot
